tests/e2e/README.md

```diff
@@ -173,14 +173,22 @@ Handy utility scripts for managing environment:
 
 <br>
 
+For running E2E tests on an Atomic site:
+
+`E2E_GROUP=atomic npm run test:e2e-dev -- --NODE_ENV=atomic`
```

Comment:
- `E2E_GROUP` is used to figure out which top folder to use within `tests/e2e` to fetch the .spec files we need to run
- `NODE_ENV` is here to target the right JSON file to use for the test data

The `atomic folder` was deleted in the last commit 4cdafe7272b76956965e6f12dff161f3abafa978 because I figured the remaining failing tests from `wcpay | shopper` were failing because UPE was enabled by default on the Atomic site, so I'll update that comment.